cpp/ql/test/experimental/library-tests/rangeanalysis/bitshift/BitShiftRange.ql

```diff
@@ -0,0 +1,25 @@
+import cpp
+import semmle.code.cpp.rangeanalysis.SimpleRangeAnalysis
+import experimental.semmle.code.cpp.rangeanalysis.extensions.ConstantLShiftExprRange
+import experimental.semmle.code.cpp.rangeanalysis.extensions.ConstantRShiftExprRange
+
+Expr getLOp(Operation o) {
```

Comment:
I can remove these helpers if you want, but I've found them to be useful in a number of cases, so it'd be nice to have them available in some other file so they can be reused.

Reply:
I guess the problem this predicate (and `getROp`) solves is that, while `Assignment` is also an operand (and so it has a `getAnOperand` predicate), you often distinguish between the left-hand side or the right-hand side of an `Assignment`. Is that correct?

I'll create an internal issue for this ergonomics problem. For now, I think its best to keep these predicates in this experimental test file.